From 856e5be27a186f702545b70ef5b19a11f2e1b25a Mon Sep 17 00:00:00 2001
From: Sachin N <sachi77n@gmail.com>
Date: Thu, 29 Oct 2020 01:28:34 -0700
Subject: [PATCH] Update .travis.yml

---
 .travis.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.travis.yml b/.travis.yml
index a4bb7b9..d472c85 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-  - stable
+  - '12.18.3'
